Source/WebKit/NetworkProcess/mac/NetworkConnectionToWebProcessMac.mm

```diff
@@ -60,6 +68,53 @@
 #endif
 }
 
-#endif // PLATFORM(MAC)
+#if ENABLE(LAUNCHSERVICES_SANDBOX_EXTENSION_BLOCKING)
+void NetworkConnectionToWebProcess::checkInWebProcess(CoreIPCAuditToken&& auditToken)
```

Comment:
Why is this taking an r-value reference? Doesn't seem useful based on the implementation.

Reply:
Ah, good point, fixed in latest patch.

Thanks for reviewing!